operatorconfig/moduleconfig/authorization/v1.10.0/deployment.yaml

```diff
@@ -1,4 +1,44 @@
+apiVersion: v1
```

Comment:
There's a new folder version for stateless work - `operatorconfig/moduleconfig/authorization/v2.0.0-alpha`

Reply:
Moved to v2.0.0-alpha